.github/workflows/integration-tests.yaml

```diff
@@ -23,6 +23,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install -r requirements.txt
+          pip install pytest
```

Comment:
In many of our repos, we put the dependancies for testing into a `test-requirements.txt` file. Please create this file and add `pytest` into it, and have `pip install -r test-requirements.txt` here instead